libraries/ts/src/margin/marginClient.ts

```diff
@@ -119,7 +123,7 @@ export class MarginClient {
     const instructions = ["repay", "borrow", "deposit", "withdraw"]
```

Comment:
These need to be more specific cause you can pick up all sorts of false positives in the logs
```suggestion
    const instructionLogHeaders = [
      "> Program log: Instruction: MarginRepay", 
      "> Program log: Instruction: MarginBorrow", 
      "> Program log: Instruction: Deposit", 
      "> Program log: Instruction: Withdraw"
    ]
```

Reply:
This would change in the future for pools vs other programs (when we have trading and stuff too right)

Reply:
Other programs would have different instruction names yes